interpreter/mesh/src/main/scala/io/buoyant/interpreter/MeshInterpreterInitializer.scala

```diff
@@ -32,6 +35,7 @@ object MeshInterpreterConfig {
   val DefaultRoot = Path.Utf8("default")
 
   val defaultRetry = Retry(1, 10.minutes.inSeconds)
+  val defaultWindowSizeKB = 1024
```

Comment:
I think bupingthat to 10 Mb is not unresonable and will give us greater guarantee that we will not see this problem show up oftenly

Reply:
agreed.  10 Mb default gives us more wiggle room